easybuild/framework/easyblock.py

```diff
@@ -3360,7 +3360,7 @@ def sanity_check_cuda(self, cuda_dirs=None):
 
         fail_msgs = []
         cfg_ccs = build_option('cuda_compute_capabilities') or self.cfg.get('cuda_compute_capabilities', None)
-        ignore_failures = not build_option('cuda_sanity_check_error_on_fail')
+        ignore_failures = not build_option('cuda_sanity_check_error_on_failed_checks')
```

Comment:
I think there's a few places where this appears in text output as `--*cuda-sanity-check-error-on-fail`, those would need to be updated

Reply:
Yeah, there are 10 places that need updating (7 in comments and 3 in output)

Reply:
Good point, will fix